### ISSUE TITLE
TCA survey bug

### ISSUE THREAD

**Issue by nilstk** [TCA-Team/TumCampusApp#389]
creating a survey ... scrolling down the list ... crash
```
java.lang.ArrayIndexOutOfBoundsException: length=14; index=14
	at android.support.v7.app.AlertController$AlertParams$1.getView(AlertController.java:891)
	at android.widget.AbsListView.obtainView(AbsListView.java:2346)
	at android.widget.ListView.makeAndAddView(ListView.java:1876)
	at android.widget.ListView.fillDown(ListView.java:702)
	at android.widget.ListView.fillGap(ListView.java:666)
	at android.widget.AbsListView.trackMotionScroll(AbsListView.java:5053)
	at android.widget.AbsListView$FlingRunnable.run(AbsListView.java:4601)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:858)
	at android.view.Choreographer.doCallbacks(Choreographer.java:670)
	at android.view.Choreographer.doFrame(Choreographer.java:603)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:844)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5417)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by kordianbruck** [comment]
@aserhany @mhmadayad any ideas? Is this already fixed?

**Comment by mhmadayad** [comment]
@kordianbruck fixed will push after an hour.

**Comment by aserhany** [comment]
@kordianbruck @mhmadayad : we checked that and didn't find the bug (no such errors get thrown). We had a suspicion about this one hard coded size of an array (boolean[] checkedFaculties for the spinner) in the activity - because there we assigned the size 14 which appears in the error. Now we assign the size dynamically according to the size of the fetchedFaculties from the db/server. We will open the new pull request respectively.
